lib/bundler/rubygems_integration.rb

```diff
@@ -521,7 +523,9 @@ def all_specs
       end
 
       def find_name(name)
-        Gem::Specification.find_all_by_name name
+        Gem::Specification.stubs.find_all { |spec|
```

Comment:
is there any reason `Gem::Specification.stubs_for` wouldn't be appropriate here?